pypit/armasters.py

```diff
@@ -163,8 +163,11 @@ def save_masters(slf, det, setup):
                            frametype='wave')
         # Wavelength fit
         gddict = ltu.jsonify(slf._wvcalib[det-1])
-        json_file=master_name('wave_calib', setup)
-        ltu.savejson(json_file, gddict, easy_to_read=True, overwrite=True)
+        json_file = master_name('wave_calib', setup)
+        if gddict is not None:
+            ltu.savejson(json_file, gddict, easy_to_read=True, overwrite=True)
+        else:
```

Comment:
When the user requests a "pixel" wavelength solution (i.e. specifically requests to not perform a wavelength solution) then gddict is not created.